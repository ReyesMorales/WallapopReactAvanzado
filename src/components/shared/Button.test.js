import React from 'react';
import renderer from 'react-test-renderer';
import Button from './Button';

describe('Button', () => {

it('renders the Button component correctly', () => {
  const tree = renderer.create(<Button>Click me</Button>).toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders the primary variant of Button correctly', () => {
  const tree = renderer
    .create(<Button variant="primary">Click me</Button>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
});
