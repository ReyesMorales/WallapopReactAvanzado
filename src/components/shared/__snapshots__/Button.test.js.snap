// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders the Button component correctly 1`] = `
<button
  className="sc-bgqQcB fgmEeA"
>
  Click me
</button>
`;

exports[`Button renders the primary variant of Button correctly 1`] = `
<button
  className="sc-bgqQcB cCxntC"
>
  Click me
</button>
`;
